opensrp-path/src/main/java/org/smartregister/path/activity/BaseRegisterActivity.java

```diff
@@ -148,6 +133,43 @@ public void onDrawerOpened(View drawerView) {
                 updateFromServer();
             }
         }
+
+        offlineMapDownloadUpdatesReceiver = new BroadcastReceiver() {
```

Comment:
Shouldn't this code be isolated into a class since it should also be used by the [BaseActivity](https://github.com/OpenSRP/opensrp-client-path/blob/1f6d3773cbd9cfae4191edf599d2106558d508a8/opensrp-path/src/main/java/org/smartregister/path/activity/BaseActivity.java)?

Reply:
Good